### ISSUE TITLE
Entity interaction plugin stepped twice with t = 0

### ISSUE THREAD

**Issue by jpfeltracco** [gtri/scrimmage#131]
Noticed that entity interaction plugins are called twice with` t = 0`. This is a bug that could cause problems if needing to run some initialization code within the step function just once and relying on the time to do so.

You can re-create by just printing out `t` in any entity interaction plugin and watching the repeated 0.

```cpp
bool SomeEntityInteraction::step_entity_interaction(std::list<sc::EntityPtr> &ents,
                                                    double t, double dt) {
  std::cout << "Step Entity Inter " << t << std::endl;
}
```

It looks like it was introduced by fd0544c2c174ad3434d783672648bb607e93e466

**Comment by jpfeltracco** [comment]
@esquires @SyllogismRXS

**Comment by esquires** [comment]
Thanks for the issue. This should now be fixed in  7aa5adf.
